client/src/app/pages/assessment-management/assessment-settings/assessment-settings-page.tsx

```diff
@@ -274,6 +275,14 @@ const AssessmentSettings: React.FC = () => {
                             />
                           </Td>
                           <Td width={25} {...getTdProps({ columnKey: "name" })}>
+                            {questionnaire.system && (
```

Comment:
To think about:

There is a good case to be made to use a ternary instead of logical `&&` to control the falsey output.  See https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
I see what you mean, however, I don't agree that it means we should always use ternary operators, because I believe they are less readable. 
We can either do !!questionnaire.system OR Boolean(questionnaire.system) to go around it, only if they var is not already a boolean.
Since in this specific system is already guaranteed to be boolean (as Questionnaire type suggests) we can do:
`questionnaire.system && ...` safely as it will turn out to either: `false && ...` which won't render the icon (it will not render false to the screen) or `true && ...` which will render only the icon to the screen

Reply:
More or less readable is a bit subjective.

In this particular case, the logical && does make sense.  And careful with anything in javascript being guaranteed to be any specific kind of type.  JS just doesn't work that way, and TS only can do static type checking.